lib/event-emit.js

```diff
@@ -12,6 +12,8 @@ Emitter.EmitterPlace = EmitterPlace;
  * @param {Object} geojson Timestamped geojson of route.
  * @param {number} updateInterval Interval (milliseconds) at which
  * to generate location events.
+ * @param {Number} start Optional index for geojson.geometry.coordinates at
```

Comment:
move this ^ down to comment above `this.setTime = function(start)` ? (instead of Emitter)